app/models/manual.rb

```diff
@@ -277,7 +277,7 @@ def add_sections_to_manual(manual, edition, published: false)
         Section.find(manual, section_uuid, published: published)
       }
 
-      removed_sections = Array(edition.removed_section_ids).map { |section_uuid|
+      removed_sections = Array(edition.removed_section_uuids).map { |section_uuid|
```

Comment:
I was slightly surprised that the `section_uuid ` block arg didn't need renaming. Presumably it was renamed in a previous PR?